source/index.js

```diff
@@ -1,2 +1,7 @@
-require( "./angular-date-picker" );
-require( "./angular-time-picker" );
+"use strict";
+
+angular.module( "vokal.datePicker", [] )
+.directive( "datePicker", require( "./directives/date-picker" ) );
+
+angular.module( "vokal.timePicker", [] )
+.directive( "timePicker", require( "./directives/time-picker" ) );
```

Comment:
These declarations should probably be in separate files, shouldn't they? To permit loading one or the other and not both, presumably.
@jrit @Tathanen